db/migrate/20180111162442_move_pictures_blobs_to_pictures.rb

```diff
@@ -0,0 +1,51 @@
+class MovePicturesBlobsToPictures < ActiveRecord::Migration[5.0]
+  class BinaryBlob < ActiveRecord::Base
+    has_many :binary_blob_parts, -> { order(:id) }, :dependent => :delete_all
+    belongs_to :resource, :polymorphic => true
+    def data
+      binary_blob_parts.pluck(:data).join
+    end
+  end
+  class BinaryBlobPart < ActiveRecord::Base; end
+  class Picture < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+  end
+
+  def up
+    add_column :pictures, :content, :binary
+    add_column :pictures, :extension, :string
+    add_column :pictures, :md5, :string
+
+    say_with_time("Moving picture content from BinaryBlobs to the pictures table") do
+      BinaryBlob.where(:resource_type => "Picture").all.each do |blob|
```

Comment:
Let's change this to a find_each just to ensure we batch it properly just in case there are lot of these.

Reply:
Also add a `.includes(:binary_blob_parts)` to avoid an N+1?

Reply:
Also also do you need `in_my_region`?  Even though they are not replicated, someone *might* have enabled that.